edb/schema/functions.py

```diff
@@ -516,6 +509,28 @@ def canonicalize_attributes(
         return s_types.materialize_type_in_attribute(
             schema, context, self, 'type')
 
+    def compile_expr_field(
+        self,
+        schema: s_schema.Schema,
+        context: sd.CommandContext,
+        field: so.Field[Any],
+        value: s_expr.Expression,
+        track_schema_ref_exprs: bool=False,
+    ) -> s_expr.CompiledExpression:
+        if field.name == 'default':
+            return value.compiled(
+                schema=schema,
+                as_fragment=True,
+                options=qlcompiler.CompilerOptions(
+                    schema_object_context=self.get_schema_metaclass(),
+                    apply_query_rewrites=not context.stdmode,
+                    track_schema_ref_exprs=track_schema_ref_exprs,
```

Comment:
I'm not sure what all of these options do, but was a drop of modalises intentional?